Source/WebKit/UIProcess/Cocoa/UserMediaPermissionRequestManagerProxy.mm

```diff
@@ -192,6 +197,14 @@ -(void)stop:(const String&)persistentId {
 }
 
 #if ENABLE(MEDIA_STREAM) && HAVE(AVCAPTUREDEVICEROTATIONCOORDINATOR)
+bool UserMediaPermissionRequestManagerProxy::isMonitoringCaptureDeviceRotation(const String& persistentId)
+{
+    if (!m_objcObserver || persistentId.isEmpty())
```

Comment:
could probably just test for perstistentId.

calling [nil blah] will return nil